docs/src/examples/components/Dropdown/State/DropdownExampleLoading.shorthand.tsx

```diff
@@ -5,12 +5,12 @@ const inputItems = ['Bruce Wayne', 'Natasha Romanoff', 'Steven Strange', 'Alfred
 
 const DropdownExampleLoading: React.FC<{ knobs: { loading: boolean } }> = ({ knobs }) => (
   <Dropdown
-    loading={knobs.loading}
-    loadingMessage="Loading..."
+    search
     multiple
     items={inputItems}
     placeholder="Start typing a name"
-    search
+    loading={knobs.loading}
+    loadingMessage="Loading..."
   />
```

Comment:
Can you please clarify why need this change? ❓

Reply:
to have consistency in the way we organize props, but I think I made a mistake here because the example is about the loading prop so it makes sense to have it first, let me revert, thanks!

Reply:
fixed